Library/Homebrew/version.rb

```diff
@@ -179,6 +179,19 @@ def self.detect(url, specs)
     end
   end
 
+  # TODO add tests if method is going to be left
+  def self.parse_version(val)
```

Comment:
@vladshablinsky I still find it's confusing to have both this method and existing `parse`. How about we name this method to `create` as `Version.create(foo)`?